accord-core/src/test/java/accord/utils/SimpleBitSetTest.java

```diff
@@ -167,9 +234,27 @@ public int nextSetBit(int fromIndex)
         @Override
         public int getSetBitCount()
         {
+            Assertions.assertThat(sut.getSetBitCount())
+                      .describedAs(toBinaryString())
+                      .isEqualTo(model.cardinality());
             return sut.getSetBitCount();
         }
 
+        public String toBinaryString()
+        {
+            StringBuilder sb = new StringBuilder();
+            for (int i = 0; i < size; i++)
+            {
+                boolean m = model.get(i);
+                boolean s = sut.get(i);
+                if (m == s)
+                    sb.append(m ? 1 : 0);
+                else
+                    sb.append("(expected=").append(m ? 1 : 0).append(", actual=").append(s ? 1 : 0).append(')');
```

Comment:
this format was hacked together to aid in debugging `setRange` issue.  Was simple and showed exactly what was needed, but open to other formats.